apps/web/tsconfig.lib.json

```diff
@@ -23,7 +23,8 @@
     "**/*.jsx",
     "**/*.ts",
     "**/*.tsx",
-    "../../apps/web/dist/dist/types/**/*.ts"
+    "../../apps/web/dist/dist/types/**/*.ts",
```

Comment:
Can we remove these two lines potentially? I think Next.js automatically adds them, but don't think we need them. Or at least "../../apps/web/dist/dist/types/**/*.ts" as we don't build to that folder (anymore).